RadarSDK/remodel/RadarBeacon.value

```diff
@@ -0,0 +1,18 @@
+# Represents a beacon.
```

Comment:
This is the description file for the auto generator. you can provide comments and nullablity in particular